### PR TITLE
Switch to Project view after creating a Flutter project

### PR DESCRIPTION
**Pull request by stevemessick**
@devoncarew As requested, switch to the Project view after creating a new Flutter project in AS.

It would be nice, I think, to autoscroll the project tree and select the file that is open in the editor. Haven't figured out how to do that yet.

### CODE REVIEW

File: flutter-studio/src/io/flutter/actions/FlutterNewProjectAction.java [hunk 1]
```diff
@@ -51,11 +61,28 @@ public void actionPerformed(AnActionEvent e) {
         }
       }
     }
-    NewProjectUtil.createNewProject(getEventProject(e), wizard);
+    createNewProject(getEventProject(e), wizard);
```
Comment (devoncarew) [inline flutter-studio/src/io/flutter/actions/FlutterNewProjectAction.java]: Perhaps a comment explaining where the the inlined `createNewProject` came from?

File: flutter-studio/src/io/flutter/actions/FlutterNewProjectAction.java [hunk 2]
```diff
@@ -51,11 +61,28 @@ public void actionPerformed(AnActionEvent e) {
...
+  private static void createNewProject(Project projectToClose, AbstractProjectWizard wizard) {
+    final boolean proceed = ProgressManager.getInstance().runProcessWithProgressSynchronously(() -> {
+      ProjectManager.getInstance().getDefaultProject(); //warm up components
+    }, ProjectBundle.message("project.new.wizard.progress.title"), true, null);
+    if (!proceed) return;
+    if (!wizard.showAndGet()) {
+      return;
+    }
+
+    final Project newProject = NewProjectUtil.createFromWizard(wizard, projectToClose);
```
Comment (devoncarew) [inline flutter-studio/src/io/flutter/actions/FlutterNewProjectAction.java]: A comment here saying that we're the `project` view (as opposed to the default `android` view) in order for Dart resources to be visible at project creation?

File: flutter-studio/src/io/flutter/actions/FlutterNewProjectAction.java (hunk 1 above)
Comment (stevemessick) [reply flutter-studio/src/io/flutter/actions/FlutterNewProjectAction.java]: Done

File: flutter-studio/src/io/flutter/actions/FlutterNewProjectAction.java (hunk 2 above)
Comment (stevemessick) [reply flutter-studio/src/io/flutter/actions/FlutterNewProjectAction.java]: Good idea, done. Also, reformat, reorganize, optimize imports. But no code changes so I'll just merge it.

**Pull request merged**